docs/sphinx/manual/launching_and_configuring_hpx_applications.rst

```diff
@@ -1497,7 +1497,7 @@ The predefined command line options for any application using
    makes the scheduler NUMA sensitive
 
 
-|hpx| configuraton options
+|hpx| configuration options
 --------------------------
```

Comment:
```suggestion
---------------------------
```

Reply:
@alexmyczko would you mind adding this change to the PR? I know it's a trivial change, but having the underline match the length of the actual heading looks prettier and makes sphinx not complain about it.